versions/latest/GAME_MASTER.json

```diff
@@ -30972,6 +30989,23 @@
       "buddyOffsetFemale": [10.0, 0.0, -80.0],
       "buddyScale": 19.0
     }
+  }, {
+    "templateId": "V0298_MOVE_BLAST_BURN",
+    "moveSettings": {
+      "movementId": 298,
```

Comment:
This needs to be fixed by updating the java proto package

Reply:
Thanks, I see that now -- I noticed its at the latest package available.

Going to be busy for the next few days but if someone doesn't beat me to it, I'll try and take a look and rebuilding the dependencies next week.

Reply:
(and I'm definitely not a java dev by any stretch, so I'm way out of the loop)